apps/api-extractor/src/analyzer/AstImportAsModule.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { AstModule } from './AstModule';
+
+export interface IAstImportAsModuleOptions {
+  readonly astModule: AstModule;
+  readonly exportName: string;
+}
+
+// TODO [MA]: add documentation
+export class AstImportAsModule {
```

Comment:
Was there a reason to introduce a new class here rather than integrate this capability into `AstSymbol`?

Reply:
Or the existing `AstImport`, which has `AstImportKind.StarImport`?